### ISSUE TITLE
Evaer Skype Video Recorder Conflict

### ISSUE THREAD

**Issue by malaguer** [dlehn/Skype-Utility-Project#22]
Is it possible to make Evaer Skype Video Recorder d3d9.dll compatible with SUP?

I remember that with older Skype versions I could manage both programs with SUP 1.3.6 version, maybe changing d3d9.dll to be 'read only' (not sure about this, time has passed...).

Now everytime I try to make both programs work together, evaer overwrites d3d9.dll with its own version and makes SUP impossible to work. And I don't know any recorder software that works currently...

Any chance to make it?

**Comment by dlehn** [comment]
I just tried it, making SUP's d3d9.dll file read only and then simply ignoring Evaer's request to restart Skype seems to work just fine (simply click the 'x' button on the popup window). No idea why the hell it's trying to delete our file, it doesn't even replace it with its own version after deleting our's, seems kinda pointless.  
Anyway, recording both audio and video seems to work fine this way, can't find any problems here.

**Comment by malaguer** [comment]
I can't see any 'X' button on the popup... This is the window as I can see it:

![2015-10-19_2053_restart_popup](https://cloud.githubusercontent.com/assets/6715879/10589472/d109f2e2-76ab-11e5-9feb-1e4018981e55.png)

I guess I can slide it to the edge of the desktop to make it 'almost' disappear. But you are totally right! Both Evaer and SUP can work together. I don't understand all this overwriting d3d9.dll thing.

Thank you so much. I can't believe the solution would be that simple.

PS. If you find out how to close that popup with the 'X? close button I'll thank you. For now I've created a script that  reduces its size to 0x0 px using CMDOW commandline and it seems to work fine, but just to know.

Anyway thanks for the tip!

**Comment by dlehn** [comment]
Well that's strange... maybe try to let it delete the file once, then just put it back with the read only property set? At least that's what I did because I wanted to see if Evaer would replace the file. Didn't check if there was an 'X' the first time, but it was definitely there after that. If that doesn't help then I have no idea.  
Anyway, glad it's at least more or less working now.

**Comment by malaguer** [comment]
Nope... I tried all the combinations before asking you for help. But the CMDOW trick is working just fine.

Maybe it depends on the OS. In my case is Windows 8.1, who knows.

Thank you again. To me this Evaer nuisance is solved. :)
